components/surveyHistorical.js

```diff
@@ -15,7 +15,7 @@ class SurveyHistorical extends Component {
 
   async componentDidMount() {
     const historicalData = Object.keys(this.state.historical)
-      .sort((a, b) => a > b)
+      .sort((a, b) => (a < b ? -1 : a > b ? 1 : 0))
```

Comment:
```suggestion
      .sort()
```
Par défaut `sort` trie déjà les éléments dans l'ordre lexicographique, du coup ça marche aussi si on enlève complètement la condition de trie (testé et effectif sur waterfox et chromium)